bin/untranslated_erb

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env ruby
+require 'rails-html-sanitizer'
```

Comment:
just noting the new behavior - because we're requiring a gem from the bundled set - this script now requires `bundle exec bin/untranslated_erb` where it used to be pure ruby.

It's not a problem, just a change (and if we have or create instructions published on how to use this to find translation opportunities, that needs to be included/updated).

Reply:
It works fine for me with just `bin/untranslated_erb` (`gem env` includes the correct gem path by default for me).

Reply:
Done